openapi/pacta.yaml

```diff
@@ -816,6 +952,15 @@ components:
     Language:
       type: string
       enum: *LANGUAGES
+    FileType:
+      type: string
+      enum: *FILE_TYPES
```

Comment:
YAML anchors work here, though I would expect using `$ref` to be the more idiomatic approach here, what problem were you seeing with it?

Reply:
Sorry I'm actually not sure what you mean here?

Reply:
The thing you're doing with `&THING` and `*THING` is called [YAML 'anchors' and 'aliases'](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/), which is a thing YAML does to allow one to say "take this chunk and pretend it's also here".

What I'm saying is that OpenAPI specs have a mechanism for that, separate from YAML, which is the `$ref` syntax It says broadly the same thing of "take this definition and use it here too". It looks like that's what we were using, so I'm wondering what's wrong with using '$ref' here and just having a single definition for each enum under `/components/`

Reply:
Beautiful, thank you for the explanation! Makes a ton of sense.

Reply:
Sent #100 to clean this up along with a bunch of other small stuff.